doc/telemetry.rst

```diff
@@ -97,81 +97,144 @@ Go Runtime metrics
-
-Yorc Workers & Tasks metrics
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-|              Metric Name              |                               Description                                |      Unit       | Metric Type |
-|                                       |                                                                          |                 |             |
-+=======================================+==========================================================================+=================+=============+
-| ``yorc.workers.free``                 | This tracks the number of free Yorc workers.                             | number of free  | gauge       |
-|                                       |                                                                          | workers         |             |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-| ``tasks.maxBlockTimeMs``              | This measures the highest duration since creation for all waiting tasks. | milliseconds    | timer       |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-| ``tasks.nbWaiting``                   | This tracks the number of tasks waiting for being processed.             | number of       | gauge       |
-|                                       |                                                                          | tasks           |             |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-| ``tasks.wait``                        | This measures the finally waited time for a task being processed.        | milliseconds    | timer       |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-| ``task.<DepID>.<Type>.<FinalStatus>`` | This counts by deployment and task type the final status of a task.      | number of tasks | counter     |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-| ``task.<DepID>.<Type>``               | This measures the task processing duration.                              | milliseconds    | timer       |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
+~~~~~~~~~~~~~~~~~~~~~
+
+The **method** label represents the HTTP verb.
+The **path** label corresponds to the request URL where slashes are replaced by dashes.
```

Comment:
Actually in the path label value, slashes are not replaced by dashes. I was the case previously when the path was part of the key

Reply:
In fact the endpointPath value containing slashes was used un the measurement name and is now used in the label. The documentation is not correct. Should we fix the code (remplaces / by -) or the documentation ?

Reply:
I think we should fix the documentation, as the replacement of slashes in metric names in previous version was only done to fulfill the prometheus constraint on metric names not accepting slashes